docs/templates.rst

```diff
@@ -1309,13 +1309,35 @@ The general syntax is ``<do something> if <something is true> else <do
 something else>``.
 
 The `else` part is optional.  If not provided, the else block implicitly
-evaluates into an undefined object::
+evaluates into an undefined object:
```

Comment:
(nit: the double-colon here can stay, see e.g. https://quick-sphinx-tutorial.readthedocs.io/en/latest/rst.html ("Code blocks are introduced by a double-colon and are indented"))

Reply:
The current syntax does not compile correctly, it can be seen in the production page at http://jinja.pocoo.org/docs/2.10/templates/#if-expression 

<img width="1109" alt="Screen Shot 2019-06-01 at 10 12 42 AM" src="https://user-images.githubusercontent.com/6611249/58750281-e58fbd80-8455-11e9-853f-45cef99ec953.png">